dts/st/u5/stm32u585aiixq-pinctrl.dtsi

```diff
@@ -1890,6 +1890,64 @@
 				bias-pull-up;
 			};
 
+			/* OCTOSPI */
+
+			octospi2_io7_ph12: octospi2_io7_ph12 {
+				pinmux = <STM32_PINMUX('H', 12, AF5)>;
+				slew-rate = "very-high-speed";
+			};
+
+			octospi2_io6_ph11: octospi2_io6_ph11 {
+				pinmux = <STM32_PINMUX('H', 11, AF5)>;
+				slew-rate = "very-high-speed";
+			};
+
+			octospi2_io5_ph10: octospi2_io5_ph10 {
+				pinmux = <STM32_PINMUX('H', 10, AF5)>;
+				slew-rate = "very-high-speed";
+			};
+
+			octospi2_io4_ph9: octospi2_io4_ph9 {
+				pinmux = <STM32_PINMUX('H', 9, AF5)>;
+				slew-rate = "very-high-speed";
+			};
+
```

Comment:
Mind to tell if this is something auto-generated or manually typed?

I plan to test `octo-spi`  and refine for `stm32h723xx`

Reply:
yes probably. 
However I put all pins def manually

Reply:
using the 
`python3 scripts/genpinctrl/genpinctrl.py -p ... -o ...`
script to generate the pin definitions